src/baseTokens.ts

```diff
@@ -0,0 +1,56 @@
+import axios from "axios";
+import { ERC20__factory } from "elf-contracts-typechain/dist/types/factories/ERC20__factory";
+import hre from "hardhat";
+import zip from "lodash.zip";
+import {
+  getTokenDecimalsMulti,
+  getTokenNameMulti,
+  getTokenSymbolMulti,
+} from "src/erc20";
+import { TokenTag } from "src/tags";
+import { getCurveTokenInfo } from "./curveToken";
+import { BaseTokenInfo, SimpleTokenInfo } from "./types";
+
+export const provider = hre.ethers.provider;
+
+export async function getBaseTokenInfos(
```

Comment:
comment that this either returns a 'native' token or a curve lp token.  not sure if native is the right word there, but some quick explanation as to what's going on here :)

Reply:
Done, I've use simpleTokenInfo as per the type